catalog/src/main/resources/db/migration/V13__Add_a_row_to_dssets.sql

```diff
@@ -0,0 +1 @@
+insert into ds_sets(name, default_datasource_id) values('quark_default', null);
```

Comment:
Since we plan to keep migration consistent with internal service, would this be ported to internal service too ?